samples/example_subplots.py

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+import click
```

Comment:
fair enough.  I apparently only remembered the bottom parts of that page but not the top.